packages/flutter_tools/lib/src/asset.dart

```diff
@@ -60,11 +73,16 @@ class AssetBundle {
      int result = await _validateFlutterYamlManifest(manifest);
      if (result != 0)
        return result;
+    } else {
+      // No manifest file found for this application.
+      return 0;
```

Comment:
I think this would be clearer if written as:
```dart
if (manifest == null)
    return 0;
  int result = await _validateFlutterYamlManifest(manifest);
  if (result != 0)
    return result;
```